js/foam/u2/Element.js

```diff
@@ -123,7 +123,11 @@ CLASS({
           console.warn('Missing Element: ', this.id);
           return
         }
-        e.classList[enabled ? 'add' : 'remove'](cls);
+
```

Comment:
This splitting should be done in the method that calls this method so that this method only needs to deal with one class at a time.